### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by Moxinilian**
In this PR, I:

* Removed accessibility to the showcase page. But as it may be useful in the future, I left the CSS and the HTML here. It just won't be rendered by Zola.
* Fixed a couple typos.
* Added @happenslol to the authors.
* Removed incorrect licences.
* Fixed the favicon and OpenGraph image.
* Made the konami easter egg on the News page do something meaningful and cool.

### CODE REVIEW

**Comment by Moxinilian** [comment]
I am sorry I have no idea what is going on with git. But at the same time it seems harmless.

File: src/templates/_shell.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
         integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp"
         crossorigin="anonymous">
     <link rel="stylesheet" href="{% block css %}{% endblock css %}">
-    <link rel="icon" href="/brand/logo-standard.svg">
+    <link rel="icon" href="/brand/logo-filled.svg">
```
Comment (piedoom) [inline src/templates/_shell.html:32]: What is the difference between standard and filled? The looked identical to me

**Comment by piedoom** [comment]
LGTM.  For the Easter egg, I have a similar brand of humor so perhaps it would be a good idea to get someone make sure the page is OK since it is technically publicly facing

**Comment by Moxinilian** [comment]
I tried to get inspiration from what Discord accepts to put up on their blog so I hope it should be okay.

**Review by piedoom — APPROVED** [review]

File: src/templates/_shell.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
     <meta property="og:description" content="{% block og_description %}{{ config.description }}{% endblock og_description %}">
     <meta property="og:type" content="{% block og_type %}website{% endblock og_type %}" />
     {% block current_url %}<meta property="og:url" content="{{ current_url }}" />{% endblock current_url %}
-    <meta property="og:image" content="/brand/logo-standard.svg" />
+    <meta property="og:image" content="/brand/logo-filled.png" />
     <meta property="og:image:width" content="425">
     <meta property="og:image:height" content="425">
```
Comment (topokel) [inline src/templates/_shell.html:21]: Should be 512x512 now.

File: src/templates/_shell.html (hunk 1 above)
Comment (topokel) [reply src/templates/_shell.html:32]: It (should) be that filled icon has a white interior while there is no white in the standard, the interior is transparent.

**Pull request merged**